### PR TITLE
Small changes to roles-pemitted docs in 5.193 (to master)

### PR DESCRIPTION
**Pull request by fturizo**
Added more clarification to the new `@RolesPermitted` annotation capabilities added in 5.193

### CODE REVIEW

File: documentation/payara-server/public-api/roles-permitted.adoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
 [[roles-permitted]]
 = RolesPermitted Support
 
-_Since Payara Server 4.1.2.182 and 5.182_
+_Since Payara Server & Micro 4.1.2.182 and 5.182_
```
Comment (OndroMih) [inline documentation/payara-server/public-api/roles-permitted.adoc]: This should be "Payara Micro" in full, not just "Micro", according to our branding guidelines.

File: documentation/payara-server/public-api/roles-permitted.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
-The Payara API provides a `@RolesPermitted` annotation that creates an authorisation interceptor for usage with CDI beans. This largely works in the same way as the `@RolesAllowed` common annotation, which in Java EE is primarily used by EJB.
+The Payara Public API provides a `@RolesPermitted` annotation that creates an authorisation interceptor for usage with CDI beans. This largely works in the same way as the `@RolesAllowed` common annotation, which in Java/Jakarta EE is primarily used by the EJB specification.
```
Comment (OndroMih) [inline documentation/payara-server/public-api/roles-permitted.adoc]: It should be Java EE / Jakarta EE.

File: documentation/payara-server/public-api/roles-permitted.adoc [hunk 3]
```diff
@@ -43,59 +43,86 @@ They are detailed below.
...
-| -
 |===
 
 [[dynamic-roles]]
 == Dynamic Roles
 
-The `@RolesPermitted` annotation also allows to dynamically specify security roles permitted to access methods in an application
-by using a predefined expression language (EL) as a dynamic role expression that interacts with the business 
-method's parameters annotated with `javax.inject.Named` annotation.
+_Since Payara Server & Micro 5.193_
```
Comment (OndroMih) [inline documentation/payara-server/public-api/roles-permitted.adoc]: Again, I recommend full "Payara Micro" instead of just "Micro"

**Comment by fturizo** [comment]
Changes made, please re-review @OndroMih

**Pull request merged**